packages/evm-predicates/src/generated/predicates/index.ts

```diff
@@ -1,13 +1,22 @@
 /* Autogenerated file. Do not edit manually. */
 
 import type { PredicateVersion } from '@fuel-connectors/common';
+import {
+  abi as abi1727718552630,
+  bin as bin1727718552630,
+  generationDate as generationDate1727718552630,
+} from './0x65e5877f81ba8d5bf10d301b8096c7529d11d88d9cc5bbe566d2cc1010544390';
 import {
   abi as abi1725479113004,
   bin as bin1725479113004,
   generationDate as generationDate1725479113004,
 } from './0xfdac03fc617c264fa6f325fd6f4d2a5470bf44cfbd33bc11efb3bf8b7ee2e938';
 
 export const PREDICATE_VERSIONS = {
+  '0x65e5877f81ba8d5bf10d301b8096c7529d11d88d9cc5bbe566d2cc1010544390': {
+    predicate: { abi: abi1727718552630, bin: bin1727718552630 },
+    generatedAt: generationDate1727718552630,
+  },
   '0xfdac03fc617c264fa6f325fd6f4d2a5470bf44cfbd33bc11efb3bf8b7ee2e938': {
     predicate: { abi: abi1725479113004, bin: bin1725479113004 },
```

Comment:
did the bytecode hash change? if not we dont need to generate a new version

Reply:
Great point, it didn't change. I've reverted the predicate generation